permutations.r

```diff
@@ -1,22 +1,49 @@
-Packages <- c("gtools", "readr", "tibble", "dplyr")
 
-lapply(Packages, library, character.only = TRUE)
+################################################################################
+
+# WHAT IS THIS SCRIPT FOR?
+
+# OH GOD WHAT IS ANYTHING FOR?
+
+################################################################################
+
+packages <- c("gtools", "readr", "tibble", "dplyr")
+
+lapply(packages, library, character.only = TRUE)
+
+
 
 x <- c("A","B","C","D","E","F")
-df <- permutations(n=6, r=6,v=x)
-write.csv(df,"C:\\Users\\xnystl\\Documents\\R projects\\MC-factory\\permutations.csv", row.names = FALSE)
+
+df <- permutations(n = 6, r = 6, v = x)
+
+write.csv(df, "permutations.csv", row.names = FALSE)
+
 permutations <- read_csv("permutations.csv")
 
-colnames(permutations) <-c("Stage_1","Stage_2","Stage_3","Stage_4","Stage_5","Stage_6")
-permutations$Sequence <- paste(permutations$Stage_1, permutations$Stage_2, permutations$Stage_3,permutations$Stage_4, permutations$Stage_5, permutations$Stage_6)
-write.csv(permutations,"C:\\Users\\xnystl\\Documents\\R projects\\MC-factory\\permutations.csv", row.names = FALSE)
+colnames(permutations) <- c("Stage_1","Stage_2","Stage_3","Stage_4","Stage_5","Stage_6")
+
+permutations$Sequence <-
+  paste(
+    permutations$Stage_1,
+    permutations$Stage_2,
+    permutations$Stage_3,
+    permutations$Stage_4,
+    permutations$Stage_5,
+    permutations$Stage_6
+  )
+
+write.csv(permutations, "permutations.csv", row.names = FALSE)
+
+n <- 3
```

Comment:
Generally avoid using the equal sign as an assignment operator, for the sake of clarity and consistency.

Also, again, try to use descriptive object names.

If you want to quickly type an assignment operator, you can use alt + -

That might seem like an equal number of keystrokes as <-, but it's faster to type (in my experience).